mozart/services/api_v01.py

```diff
@@ -1051,22 +1051,36 @@ class UserRules(Resource):
 
     def get(self):
         # TODO: add user role and permissions
-        _id = request.args.get('id')
+        _id = request.args.get("id", None)
+        _rule_name = request.args.get("rule_name", None)
         user_rules_index = app.config['USER_RULES_INDEX']
 
         if _id:
             rule = mozart_es.get_by_id(index=user_rules_index, id=_id, ignore=404)
-            if rule['found'] is False:
+            if rule.get("found", False) is False:
                 return {
                     'success': False,
                     'message': rule['message']
                 }, 404
             else:
-                rule = {**rule, **rule['_source']}
+                rule = rule["_source"]
```

Comment:
shoot my bad, i never noticed that we include all the fields when pulling all the `user_rules`
```
rule_temp = {**rule_copy, **rule['_source']}
```

we return the other fields, (`_id` , etc.) because we use that `_id` as a key when rendering the table
i think we should return similar fields when pulling individual `user_rules` as well

i think we can do something like this
```
rule = {**rule, **rule['_source']}
rule.pop('_source', None)
```
the returned `user_rule` will look like this
```
{
	"success": true,
	"rule": {
		"_index": "user_rules-grq",
		"_type": "_doc",
		"_id": "p7bamHQBCsXpnNY4AbjB",
		"_score": 2.9618306,
		"workflow": "hysds-io-lightweight-echo:v1.0.3",
		"job_spec": "job-lightweight-echo:v1.0.3",
		"priority": 1,
		"rule_name": "test-rule",
		"username": "ops",
		"query_string": "{\"match_all\":{}}",
		"kwargs": "{}",
		"job_type": "hysds-io-lightweight-echo:v1.0.3",
		"enabled": true,
		"passthru_query": false,
		"query_all": false,
		"queue": "factotum-job_worker-small",
		"modified_time": "2020-09-16T21:37:38Z",
		"creation_time": "2020-09-16T21:37:38Z",
		"tags": [
			"test-rule"
		]
	}
}
```

Reply:
sounds good @DustinKLo . I made the changes as you've requested.